postman/Event Notification - v1.9.0.postman_collection.json

```diff
@@ -129,7 +129,7 @@
 \"father.reason\": \"Father is missing!\",\n    \"mother.name\": { \"firstname\": \"{{$randomFirstName}}\", \"surname\": \"{{$randomLastName}}\" },\n    \"mother.dob\": \"1989-07-22\",\n    \"mother.nationality\": \"FAR\",\n    \"mother.idType\": \"NONE\",\n    \"mother.address\": {\n      \"country\": \"FAR\",\n      \"province\": \"5b607555-e83d-49f5-a681-d9a4776f70c5\",\n      \"district\": \"d71605e6-b20e-4b2e-a6ff-4cd2641fb0cb\",\n      \"urbanOrRural\": \"URBAN\",\n      \"town\": null,\n      \"residentialArea\": null,\n      \"street\": null,\n      \"number\": null,\n      \"zipCode\": null,\n      \"village\": null,\n      \"state\": null,\n      \"district2\": null,\n      \"cityOrTown\": null,\n      \"addressLine1\": null,\n      \"addressLine2\": null,\n      \"addressLine3\": null,\n      \"postcodeOrZip\": null,\n      \"addressType\": \"DOMESTIC\"\n    },\n    \"informant.relation\": \"GRANDMOTHER\",\n    \"informant.name\": { \"firstname\": \"{{$randomFirstName}}\", \"surname\": \"{{$randomLastName}}\" },\n    \"informant.dob\": \"1964-05-04\",\n    \"informant.nationality\": \"FAR\",\n    \"informant.idType\": \"NONE\",\n    \"informant.address\": {\n      \"country\": \"FAR\",\n      \"province\": \"5b607555-e83d-49f5-a681-d9a4776f70c5\",\n      \"district\": \"d71605e6-b20e-4b2e-a6ff-4cd2641fb0cb\",\n      \"urbanOrRural\": \"URBAN\",\n      \"town\": null,\n      \"residentialArea\": null,\n      \"street\": null,\n      \"number\": null,\n      \"zipCode\": null,\n      \"village\": null,\n      \"state\": null,\n      \"district2\": null,\n      \"cityOrTown\": null,\n      \"addressLine1\": null,\n      \"addressLine2\": null,\n      \"addressLine3\": null,\n      \"postcodeOrZip\": null,\n      \"addressType\": \"DOMESTIC\"\n    },\n    \"informant.email\": \"olivia.johnson@opencrvs.org\",\n    \"child.name\": { \"firstname\": \"{{$randomFirstName}}\", \"surname\": \"{{$randomLastName}}\" },\n    \"child.gender\": \"male\",\n    \"child.dob\": \"2024-02-02\",\n    \"child.reason\": \"Late registration reason here!\",\n    \"child.placeOfBirth\": \"PRIVATE_HOME\",\n    \"child.address.privateHome\": {\n      \"country\": \"FAR\",\n      \"province\": \"5b607555-e83d-49f5-a681-d9a4776f70c5\",\n      \"district\": \"d71605e6-b20e-4b2e-a6ff-4cd2641fb0cb\",\n      \"urbanOrRural\": \"URBAN\",\n      \"town\": \"My town\",\n      \"residentialArea\": \"My area\",\n      \"street\": \"My street\",\n      \"number\": \"11\",\n      \"zipCode\": \"10000\",\n      \"village\": null,\n      \"state\": null,\n      \"district2\": null,\n      \"cityOrTown\": null,\n      \"addressLine1\": null,\n      \"addressLine2\": null,\n      \"addressLine3\": null,\n      \"postcodeOrZip\": null,\n      \"addressType\": \"DOMESTIC\"\n    },\n    \"child.attendantAtBirth\": \"MIDWIFE\",\n    \"child.birthType\": \"SINGLE\",\n    \"child.weightAtBirth\": 4.5\n  },\n  \"annotation\": {\n    \"review.comment\": \"Comment about birth\"\n  }\n}\n"
```

Comment:
`review.signature` not in payload - not required anymore?

Reply:
Yep, its not supported on notification requests

Reply:
https://opencrvsworkspace.slack.com/archives/C07NQAULENB/p1750317228266779?thread_ts=1750314536.629609&cid=C07NQAULENB